code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -15,23 +15,8 @@
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
 
 /mob/living/simple_animal/hostile/megafauna/death(gibbed)
-	if(health > 0)
-		return
-	else
-		feedback_set_details("megafauna_kills","[initial(name)]")
-		..()
-
-/mob/living/simple_animal/hostile/megafauna/gib()
```

Comment:
>That's a wand of death.
This deadly wand overwhelms the victim's body with pure energy, slaying them without fail.
It is a small item.
It has the magic crystal shard installed.
Has 1 charge remaining.

Has 1 charge remaining. And this is a freshly spawned WoD.